modules/core/src/test/java/org/apache/ignite/internal/processors/compute/ComputeGridMonitorTest.java

```diff
@@ -440,6 +462,58 @@ public ComputeFullWithWaitTask(long timeout) {
         }
     }
 
+    /**
+     * Representa a long-running task that spawns another task.
+     */
+    @GridInternal
+    private static class InternalLongRunningTask extends ComputeTaskAdapter<Void, Void> {
+        /** Latch to indicate that job is started. */
```

Comment:
```suggestion
        /** Latch to indicate that some first job is started locally. */
```

Reply:
There is only one job that is mapped to the coordinator node.